Vizzuality/half-earth-v3
# adds sea mammals and fishes to species chart

## Substitute this line for a meaningful title for your changes
### Description
- Adds data of sea mammals and fishes to species data chart.
- Updates legend according to new data.

![image](https://user-images.githubusercontent.com/999124/165579005-9dad186c-c70f-43b3-89f5-8d845d83122c.png)


### Designs
_Link to the related design prototypes (if relevant)_
### Testing instructions
Go to [this URL](http://localhost:3000/nrc/BRA?globe=%7B%22center%22%3A%5B-54.398754351074615%2C-15.121042836690911%5D%2C%22zoom%22%3A4.816064929542014%7D), scroll down in the sidebar until you see the species chart.
### Feature relevant tickets
https://vizzuality.atlassian.net/browse/HE-352
